terraform/modules/front_door/waf.tf

```diff
@@ -0,0 +1,164 @@
+provider "aws" {
+  alias  = "north-virginia"
+  region = "us-east-1"
+
+  assume_role {
+    role_arn = var.provider_role_arn
+  }
+}
+
+resource "aws_wafv2_web_acl" "this" {
+  #checkov:skip=CKV2_AWS_31:TODO CLDC-2781 setup WAF logging in cloudwatch
+  name        = var.prefix
+  description = "Web ACL to restrict traffic to CloudFront"
+  provider    = aws.north-virginia
+  scope       = "CLOUDFRONT"
+
+  default_action {
+    allow {}
+  }
+
+  visibility_config {
+    cloudwatch_metrics_enabled = true
+    metric_name                = "waf-metrics"
+    sampled_requests_enabled   = false
+  }
+
+  rule {
+    name     = "AWSManagedRulesAmazonIpReputationList"
+    priority = 1
+
+    override_action {

```

Comment:
Looks unnecessary but you have to provide this or an `action` block as noted [here](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/wafv2_web_acl.html#rule-block)